test/test_enforced_encryption.cpp

```diff
@@ -436,54 +479,85 @@ class TestEnforcedEncryption
     const int s_yes = 1;
     const int s_no  = 0;
 
-    const bool          m_is_tracing = false;
+    const bool          m_is_tracing = true;
```

Comment:
Please disable tracing for the final CI version. Tracing is mainly intended for development stage.
```suggestion
    const bool          m_is_tracing = false;
```

Reply:
Damn. Forgot.